step-definitions/standard-ui/form.js

```diff
@@ -179,3 +203,18 @@ Then('Input fields with labels {string} should be visible',
     client.useCss();
   }
 )
+
+/**
+ * Click on label next to the evan checkbox control
```

Comment:
- docblock is also outdated ;) 
- I just saw that the function is only used for the TOS checkbox. The usual label click is not working here because auf the link in the label right? BC it also has a label `I agree with the Terms of Use.`
Maybe we can improve the label-click function?

Reply:
It's generally for all new checkboxes, because the label is used as styled overlay that lays over the original checkbox.